### PR TITLE
Outdoor v2 Style Added

### PR DESCRIPTION
**Pull request by Focusedplumes**
Style started

- Bootstrap styled started for text and cards added for the plants
- Rows being worked on for plant cards

### CODE REVIEW

**Comment by Focusedplumes** [comment]
Got 5 commits from 2 days and the one from tonight.  The one call line up is the one from tonight. Had closed it previously but was just verifying all where there, thanks!

**Comment by Focusedplumes** [comment]
Styled page.
Using bootstrap card-group for the plants cards.
Styled text and updated a little.

**Comment by Focusedplumes** [comment]
Updated Outdoor page file structure so I could add it to the button on the main page. Since my version was a little behind I also manually copied over into my branch's files the Indoor page to match what is current. This way I could add it to its button too so that lines up :).

File: .~lock.Mapping.odt#
Comment (karapk) [inline .~lock.Mapping.odt#:1]: To be deleted

File: .~lock.day1Notes.odt#
Comment (karapk) [inline .~lock.day1Notes.odt#:1]: To be deleted

File: Mapping.odt
Comment (karapk) [inline Mapping.odt:1]: To be deleted

File: starter-template/prisma/dev.db
Comment (karapk) [inline starter-template/prisma/dev.db:1]: To be deleted

File: starter-template/postcss.config.js
Comment (karapk) [inline starter-template/postcss.config.js:1]: To be deleted

File: starter-template/package.json
Comment (karapk) [inline starter-template/package.json:1]: To be deleted

File: starter-template/package-lock.json
Comment (karapk) [inline starter-template/package-lock.json:1]: To be deleted

File: day1Notes.odt
Comment (karapk) [inline day1Notes.odt:1]: To be deleted

**Review by karapk — COMMENTED** [review]
Looks good just delete the .json and the other files that are modules to avoid generating numerous conflicts.

**Pull request closed without merging**